src/samples/Textures/Textures.cpp

```diff
@@ -277,7 +277,7 @@ int mainImpl(int argc, char** argv)
     scoped.schema.scenes.scenes[0].parameters[1].options = nullptr;
     scoped.schema.scenes.scenes[0].parameters[1].dmxOffset = -1; // Auto
     scoped.schema.scenes.scenes[0].parameters[1].dmxType = RS_DMX_16_BE;
-    scoped.schema.scenes.scenes[0].parameters[0].flags = REMOTEPARAMETER_NO_SEQUENCE;
```

Comment:
flags was set for parameters index 0, but it should be set for index 1 here.
On line 291, it should be set for index 2.